02/had.py

```diff
@@ -0,0 +1,16 @@
+#----- HAD A VELBLOUD
+import sys
+
+variable = sys.argv[1]
+
+def exchange_2letters(string, position, letter):
```

Comment:
Název funkce mi připadá zavádějící. Exchange 2 letters mi evokuje vzájemnou výměnu dvou písmen. Prosím změň ho například na:
```suggestion
def replace_2letters_with_one(string, position, letter):
```

Reply:
Změněno.